docs/users/xhprof-profiling.md

```diff
@@ -1,6 +1,6 @@
 ## Profiling with xhprof
 
-DDEV-Local has built-in support for [xhprof](https://www.php.net/manual/en/book.xhprof.php). The official PECL xhprof extension does not support PHP5.6, but only PHP7.*and PHP8.*.
+DDEV-Local has built-in support for [xhprof](https://www.php.net/manual/en/book.xhprof.php). The official PECL xhprof extension does not support PHP5.6, but only PHP7 *and PHP8.*.
```

Comment:
Actually it was supposed to be like this but was a markdown error. Thanks for catching it. Does this work for you?
```suggestion
DDEV-Local has built-in support for [xhprof](https://www.php.net/manual/en/book.xhprof.php). The official PECL xhprof extension does not support PHP5.6, but only PHP 7.\* and PHP 8.\*.
```

Reply:
If so you can just commit the suggestion.